model/project_parser.go

```diff
@@ -517,35 +501,40 @@ func (pss *parserStringSlice) UnmarshalYAML(unmarshal func(interface{}) error) e
 
 // LoadProjectForVersion returns the project for a version, either from the parser project or the config string.
 // If read from the config string and shouldSave is set, the resulting parser project will be saved.
-func LoadProjectForVersion(v *Version, id string, shouldSave bool) (*Project, *ParserProject, error) {
+func LoadProjectForVersion(v *Version, id string, shouldSave bool) (*Project, *ParserProject, *ProjectConfigs, error) {
```

Comment:
This return is getting messy; we may want to refactor this to use ProjectInfo or something instead, but this PR is already giant so if you want to make a new ticket that would make sense

Reply:
Will create ticket to track this

Reply:
https://jira.mongodb.org/browse/EVG-15923